extensions/llama/python/llama.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any
+from aiconfig import Output, Prompt
+from aiconfig.Config import AIConfigRuntime
+from aiconfig.default_parsers.parameterized_model_parser import ParameterizedModelParser
+from aiconfig.model_parser import InferenceOptions
+from aiconfig.schema import ExecuteResult
+
+from llama_cpp import CreateCompletionResponse, Llama
+
+from aiconfig.util.params import resolve_prompt
+
+
+SUPPORTED_MODELS = ["llama-2-7b-chat"]
+
+
+class LlamaModelParser(ParameterizedModelParser):
+    def __init__(self, model_path: str) -> None:
+        super().__init__()
+        self.model_path = model_path
+
+    async def serialize(
+        self,
+        prompt_name: str,
+        data: Any,
+        ai_config: AIConfigRuntime,
+        parameters: dict | None = None,
+        **kwargs,
+    ) -> Prompt:
+        parameters = parameters or {}
+        out = Prompt(name="TODO", input="TODO")
+        return out
+
+    async def deserialize(
+        self, prompt: Prompt, aiconfig: AIConfigRuntime, params: dict | None = None
+    ) -> dict:
+        out = {}
+        return out
+
+    def id(self) -> str:
+        return "LLaMA"
```

Comment:
nit: keep lowercase

Reply:
@jonathanlastmileai  I can make this change upstream in ts if you agree on lowercase?

Reply:
Just wanted to double-check here r.e. ModelParser ids; I had used LLaMA from the perspective that, if this were a model provided in LastMile, its name would probably be that. It also looks like ChatGPT and HuggingFaceTextGenerationParser are other examples of capitalized ids for other 'generalized' model parsers that support underlying request models in lower case (e.g. gpt-4, llama-2-7b-chat)

Reply:
no opinion